slack_sdk/web/async_client.py

```diff
@@ -2992,6 +2997,166 @@ async def files_upload(
             kwargs["content"] = content
             return await self.api_call("files.upload", data=kwargs)
 
+    async def files_upload_v2(
+        self,
+        *,
+        # for sending a single file
+        filename: Optional[str] = None,  # you can skip this only when sending along with content parameter
+        file: Optional[Union[str, bytes, IOBase]] = None,
+        content: Optional[Union[str, bytes]] = None,
+        title: Optional[str] = None,
+        alt_txt: Optional[str] = None,
+        snippet_type: Optional[str] = None,
+        # To upload multiple files at a time
+        upload_files: Optional[List[Dict[str, Any]]] = None,
```

Comment:
Nit: Upload is already in the method name, would it make sense to call this parameter `multiple_files` or `files`

Reply:
@srajiang Good point. The reason why I hesitated to name this as `files` is that we have `file` for the single file upload and the meaning of `file` is quite different. `file` is a file content and it does not include its metadata such as alt_txt and so on. Thus, naming the list of dict objects as `files` can sound inconsistent. Thoughts?

Reply:
Perhaps just switching the parts around to `file_uploads` then? More suggestive that there's content and possibly metadata as well and still very clear that it's plural and different from `file`.

Reply:
@srajiang This makes sense! I've renamed the argument ✅

Reply:
Also, renamed the same part in Java SDK PR: https://github.com/slackapi/java-slack-sdk/pull/1065